conf/core/ConfigOptions.asciidoc

```diff
@@ -684,6 +710,14 @@ If true, will not break apart features crossing the convert bounds. This is anal
 crop.keep.entire.features.crossing.bounds used with the crop command, but used with the convert
 command only and kept as a separate option with a different default value.
 
+=== convert.bounding.box.keep.immediately.connected.ways.outside.bounds
+
+* Data Type: bool
+* Default Value: `false`
+
+If enabled, bounded queries will also return ways that are outside of `convert.bounding.box` but
+immediately connected to ways crossing or within the bounds.
+
 === convert.bounding.box.keep.only.features.inside.bounds
```

Comment:
Used in conjunction with `apidb.reader.tag.immediately.connected.out.of.bounds.ways` described above. Still needs to be implemented for `OsmXmlReader` and `OsmJsonReader` and would depend upon their source data being larger than the AOI bounds.